Add listing with image upload

sanjays442 opened the pull request:
Created a Add Listing page with Image upload

sanjays442 committed:
Added Add Listing page with signup api integration

sanjays442 committed:
Added Add Listing page with Image Upload

ravenzheng commented on app/nbproject/private/private.properties:
```diff
@@ -0,0 +1,2 @@
+index.file=index.php
```
again? Update .gitignore file please !

ravenzheng commented on app/addListing/ctrl.js:
```diff
@@ -0,0 +1,22 @@
+function ctrl($scope, $routeParams, service) {
```
Try to avoid using $scope.  In the template of a component, you can use `$ctrl` to access models.

ravenzheng commented on app/addListing/service.js:
```diff
@@ -0,0 +1,81 @@
+function service($http, endPoint, UserService) {
+    return {
+        addTreatmentCenterSignUp: function ($scope, auth, formData) {
```
Do not pass `$scope` around. Give the method what it needs, give no more than that.

sanjays442 committed:
Added nbproject folder to git ignore list

sanjays442 committed:
Removed the  from Add Listing page

sanjays442 committed:
Removed the scope variable from Add Listing page

sanjays442 committed:
Merge branch 'master' into addListingWithImageUpload

sanjays442 committed:
Remove the repeated code

Removed the repeated line .directive('fileModel', require('./fileModel')) code from components/index.js

sanjays442 committed:
Removed the repeated code

Removed line require('./featuredTreatmentCenter'), from code in app/index.js
